ainow/templates/ainow/base.html

```diff
@@ -110,7 +110,7 @@ <h2 class="mysoc-footer__site-name">TICTeC</h2>
                 <nav class="mysoc-footer__links">
                     <ul>
                         <li role="presentation"><a href="/">About</a></li>
-                        <li role="presentation"><a href="https://www.mysociety.org/files/2018/12/tictec-guide.pdf">Guide to TICTeC</a></li>
+                        <li role="presentation"><a href="{% static 'guide-2019.pdf' %}>Guide to TICTeC</a></li>
```

Comment:
This is missing a `"` after the `%}`, which means the link isn't actually appearing in the footer!